consumers/writers/influxdb/consumer_test.go

```diff
@@ -262,11 +269,17 @@ func TestSaveJSON(t *testing.T) {
 		},
 	}
 	for _, tc := range cases {
+
+		err := resetBucket()
+		require.Nil(t, err, fmt.Sprintf("Cleaning data from InfluxDB expected to succeed: %s.\n", err))
+
 		err = repo.Consume(tc.msgs)
 		assert.True(t, errors.Contains(err, tc.err), fmt.Sprintf("%s expected %s, got %s", tc.desc, tc.err, err))
-		//time.Sleep(10 * time.Second)
-		count, err := queryDB(rowCount)
-		assert.Nil(t, err, fmt.Sprintf("Querying InfluxDB to retrieve data expected to succeed: %s.\n", err))
-		assert.Equal(t, streamsSize, count, fmt.Sprintf("Expected to have %d messages saved, found %d instead.\n", streamsSize, count))
+
+		if err == nil {
```

Comment:
You can probably inline with the previous error: `if err = repo.Consume(tc.msgs); err == nil {`